e2e/cypress/integration/crash_spec.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+/// <reference path="../support/index.d.ts" />
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import '@testing-library/cypress/add-commands';
+
+import * as TIMEOUTS from '../fixtures/timeouts';
+
+/**
+ * Note : This test requires the demo plugin tar file under fixtures folder.
+ * Download :
+ * make dist latest master and copy to ./e2e/cypress/fixtures/com.mattermost.demo-plugin-0.9.0.tar.gz
+ */
+
+describe('Integrations', () => {
+
+    const pluginIdDemo = 'com.mattermost.demo-plugin'
+    const demoFile = 'com.mattermost.demo-plugin-0.9.0.tar.gz';
```

Comment:
Maybe these constants are good to be at `cypress.json`, especially the `demoFile` since it may change in the future.

Reply:
done!  [`e5d4e3e` (#130)](https://github.com/mattermost/mattermost-plugin-demo/pull/130/commits/e5d4e3ee21386c3b9af4c1db390e364683beefa7)